crates/daphne/src/vdaf/prio3_latest.rs

```diff
@@ -0,0 +1,500 @@
+// Copyright (c) 2022 Cloudflare, Inc. All rights reserved.
```

Comment:
It is the year of our lord 2024

```suggestion
// Copyright (c) 2024 Cloudflare, Inc. All rights reserved.
```